core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -201,6 +204,10 @@ private FileSystemContext(ClientContext ctx) {
         ThreadFactoryUtils.build("metrics-master-heartbeat-%d", true));
     mClosed = new AtomicBoolean(false);
 
+    mWorkerGroup = NettyUtils.createEventLoop(NettyUtils.getUserChannel(ctx.getConf()),
+        ctx.getConf().getInt(PropertyKey.USER_NETWORK_NETTY_WORKER_THREADS),
+        "netty-client-worker-%d", true);
```

Comment:
Should the name of the threadpool be keyed on something from the configuration?

Reply:
I haven't modified the code. This is they way we had it before the removal of the singleton configuration. I'm not sure what the implications are of changing the name of the threadpool

Reply:
Yeah its possible we should have an id for the context, since just knowing which threads are related to each other is a sizable benefit. I'm fine with leaving this for later as well if we want to come up with a better solution for identifying contexts (ie. makes sense to the user rather just being a random number).

Reply:
One other simple solution I thought of: client The FileSystemContext contains an AppId which is used to identify a client in metrics. How would you feel about using that as a unique identifier?

Reply:
Good idea!